src/platform/site-wide/sass/merger.scss

```diff
@@ -12,6 +12,12 @@
     min-width: 102px;
   }
 
+  .help-dropdown {
+    p {
+      line-height: .8;
+    }
+  }
+
```

Comment:
Any chance we could borrow the styles from Vets.gov, and keep this shame file from growing? 

![image](https://user-images.githubusercontent.com/1915775/47221486-7790fd00-d382-11e8-923c-3b42be96e42d.png)